jetty-core/jetty-server/src/main/config/etc/jetty-dos.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="insertHandler">
+    <Arg>
+      <New id="DosHandler" class="org.eclipse.jetty.server.handler.DoSHandler">
+        <Arg name="getId">
+          <Get>
+            <Name><Property name="jetty.dos.id.type" default="ID_FROM_REMOTE_ADDRESS"/></Name>
+            <Class><Property name="jetty.dos.id.class" default="org.eclipse.jetty.server.handler.DoSHandler"/></Class>
+          </Get>
+        </Arg>
+        <Arg name="rateFactory">
+          <New>
+            <Class><Property name="jetty.dos.rateControlFactory" default="org.eclipse.jetty.server.handler.DoSHandler$ExponentialMovingAverageRateControlFactory"/></Class>
+            <Arg name="samplePeriodMs" type="long"><Property name="jetty.dos.samplePeriodMs" default="-1"/></Arg>
+            <Arg name="alpha" type="double"><Property name="jetty.dos.alpha" default="-1.0"/></Arg>
+            <Arg name="maxRequestsPerSecond" type="int"><Property name="jetty.dos.maxRequestsPerSecond" default="100"/></Arg>
```

Comment:
I'd rename these properties as `jetty.dos.rateControlFactory.expMovingAvg.samplePeriodMs` etc. so they are scoped to the RateControlFactory.

Reply:
OK for `alpha`, but sample periods are likely to be common to all trackers and the `maxRequestsPerSecond` is definitely common to any tracker